modules/run-compliance-scan.adoc

```diff
@@ -36,7 +36,10 @@ If any of the associated check fails for a control, the entire control state is
 
 .Procedure
 
-. Navigate to the RHACS portal and open the compliance dashboard by selecting *Compliance* from the navigation menu.
+. Navigate to the {product-title-short} portal and open the compliance dashboard by selecting *Compliance* from the navigation menu.
+. Optional: By default, information under all standards is displayed in the compliance results. To view only information from specific standards, perform the following steps:
+.. Click *Manage Standards*.
```

Comment:
Minor point that button text has sentence case **Manage standards** although I do not know whether title case is a doc convention.

> Write all other areas (such as menu items, button text, form field labels, and empty state headings) in sentence case.

https://www.patternfly.org/v4/ux-writing/capitalization#capitalization-across-red-hat-uis

By the way, ongoing removal of UPPERCASE in 4.1 continuing in 4.2 will reveal inconsistency between title case and sentence case on classic pages for buttons and widget titles. It seems like technical audience tends to be immune to confusion from such details, so hopefully not causing a need for busy work to edit existing docs.

Reply:
Oh, thanks for catching this! We do use sentence case in docs also, but I sometimes forget since RH is the only place I've worked that's done so! :)